core/src/main/java/com/graphhopper/storage/BaseGraph.java

```diff
@@ -1062,33 +1049,6 @@ final void setEdgeId(int edgeId) {
             this.nextEdgeId = this.edgeId = edgeId;
         }
 
-        /**
-         * @return false if the edge has not a node equal to expectedAdjNode
-         */
-        final boolean init(int tmpEdgeId, int expectedAdjNode) {
```

Comment:
Moved this one level up, its hardly related to the iterator but its more like the initialization code of an edge state